packages/components/src/components/heading/heading-utils.ts

```diff
@@ -0,0 +1,61 @@
+import type { BreakpointCustomizable, TextSize } from '../../types';
+import { getHTMLElement } from '../../utils';
+
+export const HEADING_VARIANTS = [
+  'large-title',
+  'heading-1',
+  'heading-2',
+  'heading-3',
+  'heading-4',
+  'heading-5',
+  'headline-1', // deprecated
```

Comment:
Hmm, now the old and the new component got bigger and even more flexible? Is this really intended?